deploy-service/common/src/main/java/com/pinterest/deployservice/handler/PingHandler.java

```diff
@@ -16,33 +16,11 @@
 package com.pinterest.deployservice.handler;
```

Comment:
We need to be very careful about the changes here. We may sync next week on the purpose here. We have about 6 DB reads and 1 DB updates for processing every ping request. Not sure how many new DB operations are introduced. But I think we will hit scale limit if we 4x our current load.

Reply:
Would https://github.com/pinterest/teletraan/issues/500 be helpful for this case? :-)